templates/challenge.html

```diff
@@ -70,20 +70,24 @@
       line-height: 1.5;
     }
 
+    /* Hints styles */
     .hints-container {
       margin-bottom: 1.5rem;
       font-size: 13px;
     }
-
+    .hints-container #read-hints {
+      padding: 0.2em 0.8em;
+      font-size: 13px;
+    }
     .hints-container #hints-missing {
-      color: #aaa;
+      margin-top: 0.8em;
+      color: var(--secondary);
```

Comment:
Don't forget to add comment for this line, because it refers from picocss. See: https://github.com/laike9m/Python-Type-Challenges/blob/9bfa33160ad614ac41be5b6bcbd7bf7dd0a510c2/templates/challenge.html#L99

Reply:
Added.